src/style.css

```diff
@@ -0,0 +1,19 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
most elements haven't default margins and padding, especially in this task you have just divs, so you don't need to reset it

Reply:
fixed